### PR TITLE
used dict comps to shorten functions

### PR DESCRIPTION
**Pull request by Matthew94**

### CODE REVIEW

**Comment by Matthew94** [comment]
Closing it as if they found using `map` less readable then this would look like shit to them.

Saving them some time as they don't want to change the code in this area of the repo.

File: scripts/geoip_service.py [hunk 1]
```diff
@@ -66,32 +66,27 @@ def json_response(result):
...
+        result = {ip: gi.record_by_addr(ip) for ip in ips.split('+')}
     elif gc:
-        for ip in ips:
-            result[ip] = {
+        result = {
+            ip : {
                 'country_code': gc.country_code_by_addr(ip),
-                'country_name': gc.country_name_by_addr(ip),
-            }
+                'country_name': gc.country_name_by_addr(ip)
```
Comment (bsimpson63) [inline scripts/geoip_service.py]: can you add a trailing comma

**Comment by bsimpson63** [comment]
You can reopen this one if you'd like.

**Comment by Matthew94** [comment]
Oh sorry, I assume you'd find dict comps to be pretty bad.

I'll fix that issue with the trailing comma now.

**Comment by Matthew94** [comment]
Added the comma, is that what you wanted added?

**Comment by bsimpson63** [comment]
Thanks. I'll get this merged.

**Comment by bsimpson63** [comment]
:fish:

**Comment by Matthew94** [comment]
Awesome, thanks. :bow:

**Comment by bsimpson63** [comment]
Merged as 3469d62291f696c6881c93e023d143e21ccf8804.

Thanks for the contribution. If you tell me your reddit username I can give you a trophy.

**Comment by Matthew94** [comment]
It's Matthew94.

Thanks very much.

**Pull request closed without merging**